config/common/errors.go

```diff
@@ -49,8 +49,9 @@ var (
 	ErrTooManySystemdSources = errors.New("only one of the following can be set: contents, contents_local")
 
 	// mount units
-	ErrMountUnitNoPath   = errors.New("path is required if with_mount_unit is true and format is not swap")
-	ErrMountUnitNoFormat = errors.New("format is required if with_mount_unit is true")
+	ErrMountUnitNoPath     = errors.New("path is required if with_mount_unit is true and format is not swap")
+	ErrMountUnitNoFormat   = errors.New("format is required if with_mount_unit is true")
+	ErrMountPointForbidden = errors.New("forbidden mount point. Use a mount point that starts with /etc or /var")
```

Comment:
hmm if you look at all the other errors '.' tends to only be used as a decimal and not a period.

How about "forbidden mount point, use a mount point that starts with /etc or /var" ?

Reply:
my bad didn't notice i used a '.' thanks!